opt_PXL.ino

```diff
@@ -114,47 +118,57 @@ void setup() {
 void loop() {
   switch (menuItem) {
     case 1:
-      display.begin(16,2);
+      display.setTextSize(1);
+      display.setTextColor(WHITE);
+      display.setCursor(0, 8);
       display.println( F("1:File Select   "));
-      display.setCursor(0, 1);
+      display.setCursor(0, 14);
       display.println(m_CurrentFilename);
+      display.display();
       break;    
     case 2:
-      display.begin(16,2);
+      display.setTextSize(1);
+      display.setTextColor(WHITE);
+      display.setCursor(0,21);
       display.println( F("2:Brightness    "));
-      display.setCursor(0, 1);
+      display.setCursor(0, 2);
       display.println(brightness);
       if (brightness == 100) {
-        display.setCursor(3, 1);
+        display.setCursor(3, 2);
       }
       else {
-        display.setCursor(2, 1);
+        display.setCursor(3, 1);
       }
        display.println( F("%"));
+       display.display();
       break;    
     case 3:
-      display.begin(16,2);
+      display.begin(0,1);
       display.println( F("3:Init Delay    "));
-      display.setCursor(0, 1);
-      display.println(initDelay);    
+      display.setCursor(0, 2);
+      display.println(initDelay);
+      display.display();
       break;    
     case 4:
-      display.begin(16,2);
+      display.begin(0,1);
       display.println( F("4:Frame Delay   "));
-      display.setCursor(0, 1);
-      display.println(frameDelay);    
+      display.setCursor(0, 2);
+      display.println(frameDelay);
+      display.display();
       break;    
     case 5:
-      display.begin(16,2);
+      display.begin(0,1);
       display.println( F("5:Repeat Times  "));
-      display.setCursor(0, 1);
-      display.println(repeatTimes);    
+      display.setCursor(0, 2);
+      display.println(repeatTimes);
+      display.display();
       break;    
     case 6:
       display.begin(16,2);
       display.println( F("6:Repeat Delay  "));
       display.setCursor(0, 1);
-      display.println(repeatDelay);    
+      display.println(repeatDelay);
+      display.display();
       break;    

```

Comment:
These case statements make a pretty strong, well, case... to be made into functions. Maybe I could help you with that?